doc/src/manual/noteworthy-differences.md

```diff
@@ -21,6 +21,9 @@ may trip up Julia users accustomed to MATLAB:
     point numbers. Arbitrarily large integer literals are supported. As a result, some operations
     such as `2^-1` will throw a domain error as the result is not an integer (see [the FAQ entry on domain errors](@ref faq-domain-errors)
     for details).
+  * Integer literals are of type `Int` which corresponds to the machine integer type (`Int32` or `Int64`).
+    This means it will overflow, such that `2^64 == 0`. If you need larger values use another appropriate type,
```

Comment:
```suggestion
    This means that they will overflow, such that `2^64 == 0`. If you need larger values use another appropriate type,
```

to be consistent with the subject from the previous sentence "Integer literals".

Reply:
It is actually wrong, so I have rewritten  it.

```
julia> 10000000000000000000000000000000000000000 |> typeof
BigInt

julia> 100000000000000000000000000000 |> typeof
Int128

julia> 100000000 |> typeof
Int64
```